### PR TITLE
fix: GEOT-306: general styling part 4

### PR DESCRIPTION
**Pull request by ra8ga**

### CODE REVIEW

**Comment by ra8ga** [comment]
retest this please

File: packages/app-andrew/src/components/commons/NotificationsSidebar/styles.scss [hunk 1]
```diff
@@ -1,19 +1,17 @@
...
 .wrapper {
-  display: flex;
-  height: 100%;
-  padding: 20px 10px 20px 5px;
+  width: 100%;
+  transition: height 1s ease-out;
 
   &.opened {
-    width: $notificationSidebarWrapperWidth;
+    // width: $notificationSidebarWrapperHeight;
```
Comment (mariusz-kabala) [inline packages/app-andrew/src/components/commons/NotificationsSidebar/styles.scss:9]: remove comment

File: packages/ui/HamburgerIcon/index.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  additionalStyles?: string
+  onClick: () => unknown
+}> = ({ additionalStyles, onClick }) => {
+  return (
+    <div
+      className={cx(styles.wrapper, additionalStyles)}
+      onClick={onClick}
+      data-testid="hamburger-icon"
+    >
+      <div className={cx(styles.icon, styles.navIcon1)}>
```
Comment (mariusz-kabala) [inline packages/ui/HamburgerIcon/index.tsx:15]: can it be simplify somehow? it a lot of object to be calculated just for an icon

File: packages/ui/PostSingle/DeletePost/styles.scss [hunk 3]
```diff
@@ -3,43 +3,50 @@
...
+  &:hover {
+    @include buttonColorsHoverDefault();
   }
 }
 
 .wrapperConfirm {
   position: absolute;
   top: 0;
   left: 0;
+  z-index: 1;
```
Comment (mariusz-kabala) [inline packages/ui/PostSingle/DeletePost/styles.scss:22]: hmm didnt we agree to use a mixin for z-index?

**Pull request merged**